modules/integration_azure-firewall/variables-gen.tf

```
@@ -9,7 +9,7 @@ variable "heartbeat_notifications" {
 variable "heartbeat_aggregation_function" {
   description = "Aggregation function and group by for heartbeat detector (i.e. \".mean(by=['host'])\")"
   type        = string
-  default     = ""
+  default     = ".mean(by=['azure_resource_id'])"
```

Like other detectors

```suggestion
  default     = ".mean(by=['azure_resource_name', 'azure_resource_group_name', 'azure_region'])"
```